suite-native/transactions/src/components/TransactionsList/TransactionListItem.tsx

```diff
@@ -72,31 +86,27 @@ export const TransactionListItem = ({
     isFirst = false,
     isLast = false,
 }: TransactionListItemProps) => {
-    const includedCoinsCount = areTokensIncluded ? transaction.tokens.length : 0;
-
     const isTokenOnlyTransaction =
         areTokensIncluded && transaction.amount === '0' && transaction.tokens.length !== 0;
 
```

Comment:
I don't understand why wy can not use TokenTranferListItem anymore? Please clarify.

Reply:
Good point, I was confused by almost the same name of component wrapped in the TransactionListItemContainer inside TokenTranferListItem. I'll change it back and update 👍

Reply:
9e32cee8e9